tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+class ActionSelenium(argparse.Action):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    def __call__(self, parser, namespace, values, option_string=None):
+        browsers = values.split(',')
+        for browser in browsers:
+            try:
+                # Attempt to create class instance to see if everything is fine.
+                import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)
+            except Exception:
```

Comment:
Can we limit this to `ImportError`? I think the part about "corresponding WebDriver not installed" is obsolete now since there's not more attempt to instantiate it.